src/Graph.elm

```diff
@@ -1024,8 +979,9 @@ dfsTree seed graph =
         [ tree ] ->
             tree
 
-        _ ->
-            Debug.crash "dfsTree: There can't be more than one DFS tree. This invariant is violated, please report this bug."
+        tree :: _ ->
+            -- Debug.crash "dfsTree: There can't be more than one DFS tree. This invariant is violated, please report this bug."
+            tree
```

Comment:
This will quietly fail, but I don't think we can do any better.

Reply:
Yeah, you're probably right. Although you could replace this by an inifinite loop instead (also applies to the other `Debug.crash` cases), probably makes more sense.